scripts/observe_clear_sys_data.sh

```diff
@@ -3,19 +3,22 @@
 ## Run script command in below format
 ## ./scripts/get_sys_observability_data.sh --clustername default --containername wordpress --namespace wordpress-mysql --labels app=mysql
 
-OPTS=`getopt -o s: --long containername: --long clustername: --long namespace: --long labels: -n 'parse-options' -- "$@"`
+OPTS=`getopt -o s: --long req: --long containername: --long clustername: --long namespace: --long labels: --long fromsource: --long duration: -n 'parse-options' -- "$@"`
 eval set -- "$OPTS"
 while true; do
     case "$1" in
+        --req ) REQUEST="$2"; shift 2;;
         --containername ) CONTAINER_NAME="$2"; shift 2;;
         --clustername ) CLUSTER_NAME="$2"; shift 2;;
         --namespace ) NAMESPACE="$2"; shift 2;;
         --labels ) LABELS="$2"; shift 2;;
+        --fromsource ) FROM_SOURCE="$2"; shift 2;;
+        --duration ) DURATION="$2"; shift 2;;
```

Comment:
`--duration` and `--fromsource` options are not available for `observe` request, right? Can we state that in the script?